src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -657,13 +658,21 @@ public DarCollection cancelDarCollectionElectionsAsChair(DarCollection collectio
    * @param collection The DarCollection
    * @return The updated DarCollection
    */
-  public DarCollection createElectionsForDarCollection(User user, DarCollection collection) {
+  public DarCollection createElectionsForDarCollection(User user, DarCollection collection)
+      throws Exception {
     try {
       List<String> createdElectionReferenceIds = collectionServiceDAO.createElectionsForDarCollection(
           user, collection);
-      List<User> voteUsers = voteDAO.findVoteUsersByElectionReferenceIdList(
-          createdElectionReferenceIds);
+      if (createdElectionReferenceIds == null || createdElectionReferenceIds.isEmpty()) {
+        var e = new IllegalStateException(
+            "No elections were created for DAR Collection: %s".formatted(
+                collection.getDarCode()));
+        logException(e);
+        throw e;
```

Comment:
I've mentioned this before and I don't know how common it is in the duos codebase, but in general if you `throw`, you shouldn't `log`, since any `catch` which doesn't `throw` should be logging already. Typically, logging before `throw` causes the exception to be logged twice.

Reply:
If I comment out the `throw`s, this causes the `null` and `empty` tests to fail, which are the states we want to avoid.

Reply:
There's no need to remove the `throw` if that's the behavior we want, my suggestion was to either `throw` or `log` but not both, as that will likely result in the exception being logged twice. But that's assuming that the consent service is logging in the root exception handler, which may not be the case.